phing/tasks/setup.xml

```diff
@@ -202,7 +202,14 @@
 
   </target>
 
-  <target name="setup:config-import" description="Import configuration from the config directory.">
+  <target name="setup:pre-config" description="Setup before importing configuration.">
```

Comment:
I don't think this needs to be a separate target, it can just be rolled into setup:config-import.